extensions/gdx-setup/res/com/badlogic/gdx/setup/resources/android/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
             aidl.srcDirs = ['src']
             renderscript.srcDirs = ['src']
             res.srcDirs = ['res']
-            assets.srcDirs = ['assets']
+            assets.srcDirs = ['../%ASSET_PATH%']
```

Comment:
I don't see anything that replaces `%ASSET_PATH%`, and as far as I know, Gradle will only replace a String if it's in double-quotes and uses the `"../$ASSET_PATH"` syntax.

Reply:
gdx-setup replaces it when the project is generated. See `GdxSetup.java` line 353.

Reply:
Ah, I didn't know it was a gdx-setup thing. Resolved!